modules/nf-core/gt/gff3validator/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process GT_GFF3VALIDATOR"
+    script "../main.nf"
+    process "GT_GFF3VALIDATOR"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gt"
+    tag "gt/gff3validator"
+
+    test("custom-gff3-valid") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of(
+                        '##gff-version 3',
+                        'chr22\thavana\tpseudogene\t16572027\t16574637\t.\t+\t.\tID=gene:ENSG00000233995;Name=AP000547.1'
```

Comment:
Could you add this to the test-data repo so more people/modules can use it?

Reply:
Thank you for the feedback. This is a line from a gff3 file already present on nf-core/test-datasets. I only added the header line `##gff-version 3`. gff3validator is extremely strict and does not allow gff3 files without the header. nf-core/test-datasets gff3 files are otherwise valid.

Actually, there is a workaround, I can first pass the files through `gt/gff3` module which has already made into nf-core/modules. gt/gff3 is less stringent and automatically adds the header file by detecting the file formatting. I'll update this branch.

Reply:
Hi @mashehu 

Are you happy with the updated tests?